main/main.cpp

```diff
@@ -1956,9 +1956,15 @@ Error Main::setup2(Thread::ID p_main_tid_override) {
 	{
 		String display_driver = DisplayServer::get_create_function_name(display_driver_idx);
 
+		Vector2i *window_position = nullptr;
+		if (init_use_custom_pos) {
+			Vector2i position = init_custom_pos;
+			window_position = &position;
```

Comment:
A pointer to the local variable, which will be out of scope as soon as it's out of this `if` and become invalid.

Reply:
move to the same scope of its pointer.